eth/eventbatcher/event_batcher_test.go

```diff
@@ -0,0 +1,162 @@
+package eventbatcher_test
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/bloxapp/ssv/eth/eventbatcher"
+)
+
+func TestEventBatcher_BatchEvents(t *testing.T) {
```

Comment:
shouldn't we write a test that makes sure our batch size limit is enforced?

Reply:
We don't have a limit like that right now. Why do we need it? We want to process all block events in one transaction. Do you just want to forbid creating enormously big batches?